scripts/bitcode2cpp.py

```diff
@@ -44,14 +44,15 @@ def bundle_header(src, output):
             outfile.write("0x00 };\n\n")
             outfile.write(f"int {name}_length = {length};\n")
 
+ARCH_CHOICES = ['i686', 'x86', 'x86_64', 'arm', 'armv7', 'arm64', 'aarch64', 'wasm32', 'wasm64', 'xe64']
```

Comment:
Looking into this I think we should reorganize arm architectures in ispc.

- "arm" = "armv7" should be deprecated
- "aarch32" ( as armv8 in 32 execution mode) should be added.

It's a note to myself, no need to add it to PR.